crates/bevy_animation/src/keyframes.rs

```diff
@@ -254,6 +367,46 @@ pub struct MorphWeightsKeyframes {
     pub weights: Vec<f32>,
 }
 
+/// A [`KeyframeEvaluator`] for use with [`MorphWeightsKeyframes`].
+///
+/// You shouldn't need to instantiate this manually; Bevy will automatically do
+/// so.
+#[derive(Clone, Debug, Reflect)]
+pub struct MorphWeightsKeyframeEvaluator {
+    /// The values of the stack, in which each element is a list of morph target
+    /// weights.
+    ///
+    /// The stack elements are concatenated and tightly packed together.
+    ///
+    /// The number of elements in this stack will always be a multiple of
+    /// [`Self::morph_target_count`].
+    stack_morph_target_weights: Vec<f32>,
+
+    /// The blend weights and graph node indices for each element of the stack.
+    ///
+    /// This should have as many elements as there are stack nodes. In other
+    /// words, `Self::stack_morph_target_weights.len() *
+    /// Self::morph_target_counts as usize ==
+    /// Self::stack_blend_weights_and_graph_nodes`.
+    stack_blend_weights_and_graph_nodes: Vec<(f32, AnimationNodeIndex)>,
+
+    /// The morph target weights in the blend register, if any.
+    ///
+    /// This field should be ignored if [`Self::blend_register_blend_weight`] is
+    /// `None`. If non-empty, it will always have [`Self::morph_target_count`]
+    /// elements in it.
+    blend_register_morph_target_weights: Vec<f32>,
+
+    /// The weight in the blend register.
+    ///
+    /// This will be `None` if the blend register is empty. In that case,
+    /// [`Self::blend_register_blend_weight`] will be empty.
```

Comment:
Is this meant to be
```suggestion
    /// [`Self::blend_register_morph_target_weights`] will be empty.
```
? Since you've already stated
```
This will be `None` if the blend register is empty.
```
"this" being `blend_register_blend_weight`